### ISSUE TITLE
Trailing comma when replacing last key

### ISSUE THREAD

**Issue by baffles** [Faleij/json-stream-stringify#7]
If you return `undefined` for the last key in an object, a trailing comma is rendered in the output JSON.

For example:

```javascript
var s = new JsonStreamStringify({a: 12, b: 14, c: 13}, function (k, v) { if (k == 'c') return undefined; return v; });
s.pipe(process.stdout); // outputs {"a":12,"b":14,}
```

**Comment by Faleij** [comment]
@baffles Thank you for reporting this issue, it should now be fixed in 2.0.1 - which should be on npm in a few minutes.

**Comment by Faleij** [comment]
2.0.1 is now released
